src/unistd.rs

```diff
@@ -735,6 +737,29 @@ pub fn gethostname<'a>(buffer: &'a mut [u8]) -> Result<&'a CStr> {
     })
 }
 
+/// Get a CStr containing the name of the user logged in on the
```

Comment:
Mentioning `CStr` is redundant here, let's shorten this a little to "Get the name of the user logged in on the controlling terminal of the process".